.circleci/config.yml

```diff
@@ -36,7 +36,26 @@ jobs:
       - store_artifacts:
           path: /tmp/coverage
           prefix: coverage
-        
+  local_test:
```

Comment:
Why do we need to update the CI for this PR?

Reply:
This job is for running tests locally.
Since store_tests and store_artifacts will fail with `circleci local execute`.